cfme/base/ui.py

```diff
@@ -392,9 +392,10 @@ class database(Accordion):  # noqa
     @property
     def in_configuration(self):
         return (
-            self.accordions.settings.is_displayed and
-            self.accordions.accesscontrol.is_displayed and
-            self.accordions.diagnostics.is_displayed and
+            self.navigation.currently_selected == [] and
+            self.accordions.settings.is_displayed or
```

Comment:
Question: Why were these `and` statements changed to `or`? Is only a single accordion now displayed?

Reply:
As per my observations:
- ```is_displayed``` of ```ConfigurationView``` should be simple because whenever we try test cases with RBAC roles and if that user does not have the permission to settings or access-control or diagnostics then **in_configuration** gets failed.
- I encountered with this problem while automating this test case with RBAC for quota.
- Hence updated ```in_configuration``` with compulsory check of ```self.navigation.currently_selected == []``` with any of the remaining

Reply:
That makes sense, so it's dependent on the permissions of the user. Barring having a user dependent `is_displayed`, which could be very messy, I think this is a good approach.